Lexicon.md

```diff
@@ -3,6 +3,7 @@
 * __Bit__: A boolean variable $$ b \in \{ 0, 1 \} $$. Represents the answer to a yes/no question.
 * __Bit String__: A sequence of __Bits__ of arbitrary length $$ B^n := (b_k)_{k=0}^n , \  k \in \mathbb{N} $$.
 * __Vulcan Set__: The set of all __Bit Strings__ of infinite length $$ \mathbb{V} := \{ B_0^{\infty}, B_1^{\infty}, \dots, B_{\infty}^{\infty} \} , \ B_i^{\infty} \neq B_j^{\infty} $$.
+* __Information__: Anything that can be encoded into a __Bit String__.
```

Comment:
Is there anything that cannot be encoded into a bit string?

Reply:
Not that I can conjure... recall our conversation about encoding the set of all Reals.

Reply:
However one source of problems could be non-computable entities.

Reply:
Potential counterexample... imagine a library that contains 64k books in an ordered set. In order to encode one of the books as a bit string a program finds the index of the book in the ordered set and returns the number as a 16-bit string. Is the book encoded in the bit string?

Reply:
I think the situation you're describing sounds like a good encoder for the position of the book in the set, but not the contents of the books themselves. 

The whole system would need to be encoded to capture the book contents as well. This could be  done in the standard way by first encoding the contents of all the books, then the set structure (along with the 16-bit set index).

Reply:
Agreed, in that case the book isn't the thing encoded, only the index of the book in that particular library.